startup/preinit

```diff
@@ -2,7 +2,7 @@
 
 # SPDX-License-Identifier: GPL-2.0-only
 
-. /etc/init.d/installer-functions
+. /opt/xensource/installer/functions
```

Comment:
`Makefile` change required also here.

Reply:
They are in spec file, as  they original were.